pkg/storage/parameter_store.go

```diff
@@ -100,10 +100,13 @@ func (s ParameterStore) InsertParameterSet(ctx context.Context, params Parameter
 	return s.Documents.Insert(ctx, CollectionParameters, opts)
 }
 
-func (s ParameterStore) ListParameterSets(ctx context.Context, namespace string, name string, labels map[string]string) ([]ParameterSet, error) {
+func (s ParameterStore) ListParameterSets(ctx context.Context, listOptions ListOptions) ([]ParameterSet, error) {
 	var out []ParameterSet
 	opts := FindOptions{
-		Filter: CreateListFiler(namespace, name, labels),
+		Sort:   []string{"namespace", "name"},
```

Comment:
suggestion: You don't have to do this but one way we can standardize all three of these list functions and make it easier to maintain going forward is to change CreateListFilter to a function like `listOptions.ToFindOptions` and it would handle setting the sort, filter, skip and limit.

Reply:
That also crossed my mind since we write all of the same kind of input to the FindOptions to credential, parameter, and installation! Updated 🙂